### PR TITLE
Protocol changes for 1.21.30

### PR DESCRIPTION
**Pull request by dries-c**

### CODE REVIEW

**Comment by ipad54** [comment]
Why you haven't added the new packets to the packet pool registry?

**Comment by dries-c** [comment]
Very good catch, I indeed haven't. For no reason what so ever

File: src/types/camera/CameraPreset.php [hunk 1]
```diff
@@ -49,8 +53,14 @@ public function getPitch() : ?float{ return $this->pitch; }
 
 	public function getYaw() : ?float{ return $this->yaw; }
 
+	public function getRotationSpeed() : ?float { return $this->rotationSpeed; }
+
+	public function getSnapToTarget() : ?bool { return $this->snapToTarget; }
```
Comment (Superomarking) [inline src/types/camera/CameraPreset.php:58]: Usually boolean fields are prefixed with `is` or `has` (`hasSnapToTarget`)

Comment (ipad54) [reply src/types/camera/CameraPreset.php:58]: Yes, but this is not always boolean. It also can be nullable.

Comment (Superomarking) [reply src/types/camera/CameraPreset.php:58]: Alright thanks

**Review by Superomarking — APPROVED** [review]

**Review by TwistedAsylumMC — APPROVED** [review]

**Comment by dktapps** [comment]
FYI the changes to ProtocolInfo, PacketPool etc are normally done by https://github.com/pmmp/BedrockProtocolDumper.
They shouldn't need to be done by hand.

**Review by dktapps — APPROVED** [review]

**Pull request merged**